apigrpc/apigrpc.swagger.json

```diff
@@ -100,21 +133,6 @@
             "schema": {
               "$ref": "#/definitions/apiAccountCustom"
             }
-          },
```

Comment:
@zyro You'll need to manually undo this changes the omit query params in POST and DELETE methods because they're not generated correctly by Swagger codegen so we manually patch them in.

Reply:
Looks like I reverted most but not all, will fix.